### DISCUSSION TITLE
Movement 上的 AI 图片生成项目

### DISCUSSION THREAD

**Discussion by newraina** [NonceGeek/Move-Camp#330] [Projects]
一些可能的点子，先记在这里，在实现的时候再细化：

- 按量付费的图片生成 (在 rooch 上实现了一版)
- 生成后直接铸造 NFT，实现 NFT 生成 / 拥有 / 交易的一体化
- 有 Crypto 特色的 AI 艺术创作主题，如：
  - 流动性池艺术，将代币 / 流动性参数 / 波动可视化
  - 链上交易数据可视化，反映市场情绪和动态 
  - ...

**Comment by leeduckgo** [comment]
> 一些可能的点子，先记在这里，在实现的时候再细化：
> 
> * 按量付费的图片生成 (在 rooch 上实现了一版)
> * 生成后直接铸造 NFT，实现 NFT 生成 / 拥有 / 交易的一体化
> * 有 Crypto 特色的 AI 艺术创作主题，如：
>   
>   * 流动性池艺术，将代币 / 流动性参数 / 波动可视化
>   * 链上交易数据可视化，反映市场情绪和动态
>   * ...

Good one! Looking forward about it! 🤩
